### PR TITLE
Refactoring the Data component

### PR DESCRIPTION
**Pull request by The24thDS**
Multiple changes in this PR. But all of them deal with the Data component:

- Removing chartData prop because the chart is generated with the static data found inside the `SourceChart.js` so passing our actual data to the chart is useless;
- Updating the design of the SourceChart in order to have a consistent design between the 2 charts (displaying colors and percentages, using the heart rating feature implemented for the original chart);
- Proper alignment of the section.

Before:
before
After:
after

### CODE REVIEW

**Comment by MattCSmith** [comment]
Nice job on this. Although unfortunately there is a conflict due to another PR working on this section of the app. Could you kindly rectify this conflict :)

**Comment by The24thDS** [comment]
Done 😊

File: src/components/Data/SourceChart.js [hunk 1]
```diff
@@ -18,16 +18,18 @@ const data = {
   labels: ['Global Job Demand', 'US Job Demand', 'Startup Job Demand', 'Remote Job Demand'],
 };
 
-const SourceChart = () => (
+const SourceChart = ({loveFunction}) => {
+  const hearts = loveFunction(4);
+  return(
   <div>
-    <h5 className="mb-4">Love by Community: 4 / 5</h5>
+    <h5 className="mb-4">Developer Love: {hearts}</h5>
```
Comment (aneagoie) [inline src/components/Data/SourceChart.js]: Could we change this to template strings just so we are explicit in what we are doing?

````
`Developer Love: ${hearts}`
```

Comment (The24thDS) [reply src/components/Data/SourceChart.js]: Sure thing :)

**Comment by The24thDS** [comment]
All checks have passed :)

**Comment by aneagoie** [comment]
Nice work on this one! Merging 👍

**Pull request merged**